src/terminal-app.c

```diff
@@ -1730,8 +1715,7 @@ terminal_app_handle_options (TerminalApp *app,
 	GList *lw;
 	GdkScreen *gdk_screen;
 
-	gdk_screen = terminal_app_get_screen_by_display_name (options->display_name,
-	             options->screen_number);
```

Comment:
This `options->screen_number` is defined in `src/terminal-options.[h|c]`. I think it could be removed as well, as it's not used anywhere else.

Reply:
Done, thank you!